.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+# Set update schedule for GitHub Actions
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "weekly"
+  - package-ecosystem: "cargo"
+    directory: "cookbook/tests/"
+    schedule:
+      # Check for updates to Rust dependencies every week
+      interval: "weekly"
```

Comment:
NOTE to myself: I need to test this (first time using the `package-ecosystem: "cargo"`).